terraform/workspace_variables/preprod.tfvars.json

```diff
@@ -0,0 +1,25 @@
+{
+  "environment_name": "preprod",
+  "key_vault_name": "s165t01-preprod-kv",
```

Comment:
See here https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/45/files#r881427525